code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -197,6 +215,20 @@
 			download_objective.owner = owner
 			download_objective.gen_amount_goal()
 			add_objective(download_objective)
+		else if(midround && prob(30))
+			var/datum/objective/breakout/breakout_objective = new
+			breakout_objective.owner = owner
+			breakout_objective.find_target_by_role(role = ROLE_TRAITOR, role_type = 1)
+			var/datum/antagonist/traitor/T = breakout_objective.target.has_antag_datum(/datum/antagonist/traitor)
+			if(!T || locate(/datum/objective/martyr) in T.objectives)
+				//they WANT to die, so we just make this a steal instead
+				qdel(breakout_objective)
+				var/datum/objective/steal/steal_objective = new
+				steal_objective.owner = owner
+				steal_objective.find_target()
+				add_objective(steal_objective)
+			else
+				add_objective(breakout_objective)
```

Comment:
Just let martyrs have the breakout objective, it makes more sense than going guns blazing only to have the steal objective target looted post-mortem.